src/api/flaskr/command/import_user.py

```
@@ -14,20 +20,42 @@ def import_user(
+                }
+                aggregate, _ = ensure_user_for_identifier(
+                    app,
+                    provider="phone",
+                    identifier=normalized_mobile or temp_id,
+                    defaults=defaults,
+                )
         else:
-            user_id = user.user_id
+            entity = get_user_entity_by_bid(aggregate.user_bid, include_deleted=True)
+            updates = {"identify": normalized_mobile}
+            if aggregate.state == USER_STATE_UNREGISTERED:
```

Ruff (0.14.1)

Undefined name `USER_STATE_UNREGISTERED`

(F821)